.travis.yml

```diff
@@ -18,3 +18,5 @@ script:
     BUILD_IN_DOCKER=1
     BOARDS="native samr21-xpro"
     make -CRIOT/examples/hello-world buildtest
+  # run static tests
+  - docker run --rm -ti -v $(pwd)/RIOT:/data/riotbuild -e CI_BASE_BRANCH=2020.04-branch riotdocker:latest ./dist/tools/ci/static_tests.sh
```

Comment:
Why 2020.04 ?

Reply:
same as above, didn't want to change the base branch in one go. next PR already done. :)